packages/jest-resolve/src/index.ts

```diff
@@ -438,4 +434,5 @@ Please check your configuration for these entries:
   return error;
 };
 
-export = Resolver;
+export {ModuleNotFoundError};
+export default Resolver;
```

Comment:
Oh, we can't do that, as it breaks `require('jest-resolve')` use case. How about:
```ts
Resolver.ModuleNotFoundError = ModuleNotFoundError;
export = Resolver
```
That should do it, we do similar thing for `jest-get-type`

Reply:
Since `Resolver` is a class, so `Resolver.ModuleNotFoundError = ModuleNotFoundError;` will have ts error `Property 'ModuleNotFoundError' does not exist on type 'typeof Resolver'.`

About the breaking change you mentioned before, I think it may be not a breaking change since ts won't check static method like:

```js
class CustomResolver implements Resolver {
  ...
}
const CustomResolver: Resolver = {
  ...
}
```

So how do you think current change ?

Reply:
Ah, ok, that should be fine, as it's used directly from Jest's `jest-resolver`.